distributed/stealing.py

```diff
@@ -23,7 +23,7 @@
 # submission which may include code serialization. Therefore, be very
 # conservative in the latency estimation to suppress too aggressive stealing
 # of small tasks
-LATENCY = 0.1
```

Comment:
I don't think this is realistic, see https://github.com/dask/distributed/pull/5390